lib/analyzer.js

```diff
@@ -413,6 +429,20 @@ Analyzer.prototype._metadataTree = function _metadataTree(htmlMonomer,
   }.bind(this));
 };
 
+/**
+ * Calls `dom5.nodeWalk` on each document that `Anayzler` has laoded.
+ * @param  {Object} predicate A dom5 predicate.
+ * @return {Object}
+ */
+Analyzer.prototype.nodeWalkDocuments = function nodeWalkDocuments(predicate) {
+  var results = []
+  for (var href in this.parsedDocuments) {
```

Comment:
Should you use `Object.keys()` here so you don't access the prototype? Even better, can you use a `Map`?